go.mod

```diff
@@ -4,9 +4,9 @@ go 1.18
 
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
-	github.com/argoproj-labs/argocd-autopilot v0.3.0
+	github.com/argoproj-labs/argocd-autopilot v0.3.1
 	github.com/argoproj/applicationset v0.4.1
-	github.com/argoproj/argo-cd/v2 v2.3.2
+	github.com/argoproj/argo-cd/v2 v2.3.3
```

Comment:
very much not sure to which version of argo-cd I should bump (and where I should bump in argo-cd accordingly).
I see the latest argo-cd in our fork is 2.4.0.
but the cli-v2 should use a different argo-cd version than that of our fork?